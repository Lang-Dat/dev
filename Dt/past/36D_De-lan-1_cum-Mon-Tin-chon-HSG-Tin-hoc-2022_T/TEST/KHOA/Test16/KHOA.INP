67235295177623663315456584822816732748823124222518787337556753117169768967422451351418655429446811663111348844884311136611864492455681415315422476986796171135765573378781522242132884723761822848565451336632677159253276