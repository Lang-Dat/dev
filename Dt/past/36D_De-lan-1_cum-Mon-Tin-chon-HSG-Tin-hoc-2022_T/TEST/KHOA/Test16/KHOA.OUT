2
218 1
109 110
